Acceptance-Tests.md

```diff
@@ -31,7 +31,8 @@ oppia/core/tests/
     │  ├── puppeteer-utils.ts
     │  ├── show-message-utils.ts
     │  ├── test-constants.ts
-    │  └── user-factory.ts
+    │  ├── user-factory.ts
+    │  └── console-reporter.ts
```

Comment:
Add an explanation for this file inside, The directory structure is as follows... The puppeteer-testing-utilities... new bullet point.